### PR TITLE
EFF domains without HSTS

### PR DESCRIPTION
**Pull request by hstseverywhere**

### CODE REVIEW

**Comment by pipboy96** [comment]
Can you attempt to discover subdomains (using Sublist3r and Amass) and separate the ruleset into several files?

File: src/chrome/content/rules/EFF.xml [hunk 1]
```diff
@@ -1,17 +1,26 @@
-<!--
-	Other EFF rulesets:
-		- Freerangekitten.com.xml
-		- IFightSurveillance.org.xml
-		- ManilaPrinciples.org.xml
-		- Panopticlick.com.xml
+<ruleset name="EFF">
 
--->
+	<target host="freerangekitten.com" />
```
Comment (cschanaj) [inline src/chrome/content/rules/EFF.xml]: IIRC, this domain is intentionally leave unprotected, and reserved like `example.com`. And this domain is also used in the unit tests.

Comment (cschanaj) [inline src/chrome/content/rules/EFF.xml]: IIRC, this domain is intentionally leave unprotected, and reserved like example.com. And this domain is also used in the unit tests.

**Comment by J0WI** [comment]
Maybe the @EFForg wants to preload them anyway?
@zoracon @Hainish

**Comment by pipboy96** [comment]
@J0WI That would be the actual solution. We still may need to add the ruleset until they will be preloaded in all targets incl. FF ESR.

File: src/chrome/content/rules/EFF.xml [hunk 2]
```diff
@@ -1,17 +1,26 @@
-<!--
-	Other EFF rulesets:
-		- Freerangekitten.com.xml
-		- IFightSurveillance.org.xml
-		- ManilaPrinciples.org.xml
-		- Panopticlick.com.xml
```
Comment (J0WI) [inline src/chrome/content/rules/EFF.xml]: The targets added in this PR are already covered in these files. We shouldn't merge them back into one file.

File: src/chrome/content/rules/EFF.xml (hunk 1 above)
Comment (J0WI) [reply src/chrome/content/rules/EFF.xml]: https://github.com/EFForg/https-everywhere/pull/9095

**Comment by hstseverywhere** [comment]
Sorry I messed up the "thread" I think. Not used to GitHub

**Comment by pipboy96** [comment]
Your PR has been successfully merged (https://github.com/EFForg/https-everywhere/commit/60d65b37427c1680b02d9e2e3241c4e8d2662900).

**Pull request closed without merging**